dev/Repeater/APITests/ItemTemplateTests.cs

```diff
@@ -453,7 +513,7 @@ public void ValidateCustomElementFactory()
             {
                 Width = 400,
                 Height = 400,
-                ScrollViewer = new ScrollViewer()
+                ScrollViewer = new Windows.UI.Xaml.Controls.ScrollViewer()
```

Comment:
Why WUX ScrollViewer?

Reply:
because it conflicts with mux.scrollviewer